src/components/Dao/Wiki/CustomDispatcher.ts

```diff
@@ -0,0 +1,49 @@
+import { IProposalType } from "@daostack/client";
+import { GenericSchemeRegistry, Action } from "genericSchemeRegistry";
+import {
+  IDispatcherMethods,
+  IDaoInformation,
+  IProposalWikiOptions,
+  IVoteOptions,
+} from "./types";
+
+export class CustomDispatcher {
```

Comment:
can you please add comments here describing the raison d'etre of this class?

Reply:
This custom dispatcher is a way to pass alchemy's actions to the underscore protocol elements, this way, an user can create a proposal by interacting with the wiki component

It facilitates the communication between alchemy's dispatcher and underscore protocol components

Reply:
sorry, I meant to add the comment stop the code.